prac2_login_verification_UI/verification.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';

```

Comment:
This page also suffers from overflowing at the bottom. Please try to resolve it too.

You should look at ScreenUtil package in flutter.

![WhatsApp Image 2022-12-23 at 10 45 41](https://user-images.githubusercontent.com/106133014/209276058-02edf91a-cbad-4306-9422-8f8b2a34af21.jpeg)

Reply:
The problem still persists @sAchin12320-web even after your minor UI adjustments.

Please consider using ScreenUtil package to make the screen adaptive to various display sizes. :thought_balloon: